lib/faker/pokemon/en.ex

```diff
@@ -7,13 +7,285 @@ defmodule Faker.Pokemon.En do
 
   @doc """
   Returns a Pokemon name
+
+  ## Examples
+
+      iex> Faker.Pokemon.name()
```

Comment:
should be `Faker.Pokemon.En.location()`

Reply:
copy and :spaghetti: 

:laughing: 

fixed!